doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1842,20 +1842,21 @@ Number notations
      only for integers or non-negative integers, and the given number
      has a fractional or exponent part or is negative.
 
-   .. exn:: int63 are only non-negative numbers.
+   .. exn:: underflow in int63 literal @bigint
 
-      :n:`Int63.int` are unsigned integers.
+      The constant is too small to fit into a signed 63-bit integer :n:`Sint63.int`.
```

Comment:
Which of these would cause the error: `1 x 10^-20`, `-1 x 10^20` or `-10000000000000000000000000000000`?

Usually "underflow" means only the first example and the other two are "overflow".  Since this PR seems to be about integers,  I wonder whether the word choice is correct.

Some explanation here:
https://stackoverflow.com/questions/6360049/what-are-arithmetic-underflow-and-overflow-in-c

Reply:
In that case "underflow" is indeed incorrect in this context. I guess the input's sign is plausibly known to the user, so having different errors for different signs might not be necessary.